src/averageAges.js

```diff
@@ -1,31 +1,44 @@
 'use strict';
 
-function calculateMenAverageAge(people, century) {
-  const men = people.filter(person =>
-    person.sex === 'm' && (!century || Math.ceil(person.died
-      / 100) === century));
-  const totalAge = men.reduce((sum, man) =>
-    sum + (man.died - man.born), 0);
-
-  return totalAge / men.length;
+function calculateMenAverageAge(allPeople, century) {
+  function calculateAverage(people) {
```

Comment:
this func is same for `calculateMenAverageAge` and `calculateWomenAverageAge` pls move it to the global scope and reuse in both of the mentioned functions